superchain/configs/configs.toml

```diff
@@ -0,0 +1,1088 @@
+##############################################
```

Comment:
I wonder if we should just rip this out altogether?

Reply:
It'll be a bit ugly for downstream bindings to try to clobber this together without either using custom go scripts or including all static config files and patching it together.

The issue is really traversing the directory in `no_std` doesn't work nicely - we'd end up having to explicitly list out the configs in rust and embed each config file

Reply:
I can certainly play around with it in a follow-up pr, but I think this may be the best solution since all it requires is a quick `just codegen` and makes bindings 10x easier and less liable to miss config data